crates/ursa-network/src/tests/mod.rs

```diff
@@ -0,0 +1 @@
+mod service_tests;
```

Comment:
no need for mod here if linking tests directly with path

Reply:
Maybe you are referring to another case which I am not aware of. But here you def need to mod.

Reply:
Can remove the mod.rs file